src/devices/IBT_6XS_SOLIS6_json.h

```diff
@@ -0,0 +1,70 @@
+const char* _IBT_6XS_SOLIS6_json = "{\"brand\":\"Inkbird/Tenergy\",\"model\":\"iBBQ/SOLIS6\",\"model_id\":\"IBT-6X(S)/SOLIS-6\",\"condition\":[\"name\",\"index\",0,\"iBBQ\",\"&\",\"manufacturerdata\",\"=\",44,\"index\",0,\"00000000\"],\"properties\":{\"tempc\":{\"condition\":[\"manufacturerdata\",20,\"!\",\"f6ff\"],\"decoder\":[\"value_from_hex_data\",\"manufacturerdata\",20,4,true,false],\"post_proc\":[\"/\",10]},\"tempc2\":{\"condition\":[\"manufacturerdata\",24,\"!\",\"f6ff\"],\"decoder\":[\"value_from_hex_data\",\"manufacturerdata\",24,4,true,false],\"post_proc\":[\"/\",10]},\"tempc3\":{\"condition\":[\"manufacturerdata\",28,\"!\",\"f6ff\"],\"decoder\":[\"value_from_hex_data\",\"manufacturerdata\",28,4,true,false],\"post_proc\":[\"/\",10]},\"tempc4\":{\"condition\":[\"manufacturerdata\",32,\"!\",\"f6ff\"],\"decoder\":[\"value_from_hex_data\",\"manufacturerdata\",32,4,true,false],\"post_proc\":[\"/\",10]},\"tempc5\":{\"condition\":[\"manufacturerdata\",36,\"!\",\"f6ff\"],\"decoder\":[\"value_from_hex_data\",\"manufacturerdata\",36,4,true,false],\"post_proc\":[\"/\",10]},\"tempc6\":{\"condition\":[\"manufacturerdata\",40,\"!\",\"f6ff\"],\"decoder\":[\"value_from_hex_data\",\"manufacturerdata\",40,4,true,false],\"post_proc\":[\"/\",10]}}}";
+/*R""""(
+{
+   "brand":"Inkbird/Tenergy",
+   "model":"iBBQ/SOLIS6",
+   "model_id":"IBT-6X(S)/SOLIS-6",
```

Comment:
I'm worried about the parenthesis in the model id. Not sure how the different client programs handle this. Maybe we stick to IBT-6XS/SOLIS-6 ?

Reply:
Yes, I wasn't actually sure about that, so best to keep that just in the docs with the parenthesis.

Also thinking about renaming the Solis 6 to just Solis, as it is also being sold with just 2 probes, but still has all the 6 connectors for additional probes. What do you think?

Reply:
Resubmitted with just the parenthesis change.

Reply:
Yes, makes sense to rename it to Solis only